File: LarkCardKit/Models/Elements/MultiSelectPerson.cs
using System.Text.Json.Serialization;
using LarkCardKit.Models.Behaviors;

namespace LarkCardKit.Models.Elements;

public class MultiSelectPerson : Element
{
    public override string Tag => "multi_select_person";

    [JsonPropertyName("name")]
    public string? Name { get; set; }
    
    [JsonPropertyName("required")]
    public bool? Required { get; set; }
    
    [JsonPropertyName("placeholder")]
    public PlainText? Placeholder { get; set; }
    
    /// <summary>
    /// 文本标签
    /// </summary>
    [JsonPropertyName("label")]
    public PlainText? Label { get; set; }
    
    [JsonPropertyName("selected_values")]
    public List<string>? SelectedValues { get; set; }
    
    [JsonPropertyName("options")]
    public List<PersonOption>? Options { get; set; }
    
    [JsonPropertyName("width")]
    public string? Width { get; set; }
    
    [JsonPropertyName("disabled")]
    public bool? Disabled { get; set; }

    /// <summary>
    /// 禁用提示
    /// </summary>
    [JsonPropertyName("disabled_tips")]
    public PlainText? DisabledTips { get; set; }

    [JsonPropertyName("confirm")]
    public ConfirmConfig? Confirm { get; set; }
    
    [JsonPropertyName("behaviors")]
    public List<object>? Behaviors { get; set; }

    }

public class PersonOption
{
    [JsonPropertyName("value")]
    public string Value { get; set; } = string.Empty;
}

File: Python/lark_card_kit/models/elements/multi_select_person.py
"""MultiSelectPerson element for multiple person selection."""

from __future__ import annotations

from dataclasses import dataclass, field
from typing import Any

from .base import Element
from .button import ConfirmConfig
from .plain_text import PlainText


@dataclass
class MultiSelectPerson(Element):
    """Multi-person selector element for multiple person selection.

    Allows users to select multiple people from the organization.
    """

    tag: str = field(default="multi_select_person", init=False)
    name: str | None = None
    placeholder: PlainText | None = None
    selected_values: list[str] | None = None
    disabled: bool | None = None
    disabled_tips: PlainText | None = None
    confirm: ConfirmConfig | None = None
    width: str | None = None
    behaviors: list[Any] | None = None